src/TinyIoC/TinyIoC.cs

```diff
@@ -50,11 +50,8 @@
 #undef READER_WRITER_LOCK_SLIM
 #endif
 
-#if NETSTANDARD1_0 || NETSTANDARD1_1 || NETSTANDARD1_2
-#undef APPDOMAIN_GETASSEMBLIES

```

Comment:
@niemyjski - Sure, but relying on a polyfill means TinyIoC won't work out-of-the-box.